Valet.xcodeproj/project.pbxproj

```diff
@@ -497,6 +639,106 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		26E682811BA8B3F900EFF4EA /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
Also looks like we already had `CURRENT_PROJECT_VERSION = 1;` in our `Valet Mac` target. TIL.